src/tasks/actions/thunks.ts

```diff
@@ -125,7 +125,7 @@ export const getAllTasks = (name?: string) => async (
     const resp = await fetchTasks(query)
 
     let nonNormalizedTasks = resp.data.tasks
-    let next = resp.data.links.next
+    let next = resp.data.links?.next
```

Comment:
I think you'll probably also want to add a `?.` to `resp.data` in case links doesn't come back so like

```ts
resp.data?.links?.next
```

Reply:
good call